src/API/Google/Middleware.php

```diff
@@ -529,6 +529,15 @@ protected function get_site_timezone_string(): string {
 	 */
 	public function get_account_review_status() {
 		try {
+			/** @var Merchant $merchant */
+			$merchant = $this->container->get( Merchant::class );
+
+			if ( $merchant->is_standalone() ) {
+				$response = [];
+				do_action( 'woocommerce_gla_request_review_standalone', $response );
+				return $response;
```

Comment:
May I ask what the intention of this action hook is? It looks like if it will ever be useful it might be better to have this as a filter instead. Always passing an empty array to the action hook doesn't provide any benefit to the function that hook into this.

In my opinion I wouldn't have added any action hooks in the function `get_account_review_status` unless we foresee us specifically needing them for anything specific.

Reply:
Right, wanted a filter but did the action instead. I just removed it at the end.